### PR TITLE
cypress, test for default cookie

### PR DESCRIPTION
**Pull request by Stalkakuma**
Added Cypress and wrote two tests that should:

1. Check if the cookie is set
2. Check if cookie has it's value updated

### CODE REVIEW

**Review by bring-shrubbery — COMMENTED** [review]
Looks good, but you should also add a job to the Github Actions to automatically run cypress in a headless browser before merging anything.

File: .github/workflows/test.action.yml [hunk 1]
```diff
@@ -27,3 +27,8 @@ jobs:
       - run: yarn install
       - run: yarn build
       - run: yarn test
+      - name: Cypress run
```
[the remark this answers is not in the document]
Comment (Stalkakuma) [reply .github/workflows/test.action.yml:30]: Agree, working on making it "good"

[the remark this answers is not in the document]
Comment (bring-shrubbery) [reply .github/workflows/test.action.yml:30]: It's more of a "make it run faster" problem, nothing wrong with running it one after another, but having Jest tests and Cypress tests not wait for each other would make everything faster.

File: .github/workflows/test.action.yml [hunk 2]
```diff
@@ -27,3 +27,8 @@ jobs:
...
+        uses: cypress-io/github-action@v2
+        with:
+          build: yarn build
+          start: yarn start
```
Comment (bring-shrubbery) [inline .github/workflows/test.action.yml:34]: You don't have a start script in `package.json` btw. Also you don't have a script for starting cypress at all.

File: .github/workflows/cypress.tests.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+   container: cypress/browsers:node12.18.3-chrome87-ff82
+   steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+
+
+     - name: Cypress run
+       uses: cypress-io/github-action@v2
+       with:
+        browser: firefox
```
Comment (bring-shrubbery) [inline .github/workflows/cypress.tests.yml]: What I meant by "cypress should be a separate job" is what you did with the `cypress-run` keyword here. In this case, you created a separate "workflow" with a job called `cypress-run` inside of it. What I was hoping for is one workflow called `test.action.yml` with two jobs inside that run in parallel: `build` and `cypress-run`.

You can read more about the definitions here: https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow

File: .github/workflows/test.action.yml [hunk 4]
```diff
@@ -26,4 +26,25 @@ jobs:
...
+     - name: Checkout
+       uses: actions/checkout@v2
+       
+     - name: Use Node.js ${{ matrix.node-version }}
+       uses: actions/setup-node@v2
+       with:
+         node-version: ${{ matrix.node-version }}
+
+     - name: Cypress run
+       uses: cypress-io/github-action@v2
```
Comment (bring-shrubbery) [inline .github/workflows/test.action.yml:50]: Also, don't forget to always add new line at the end, GitHub will show this as this red outlined minus sign.

File: .github/workflows/test.action.yml [hunk 5]
```diff
@@ -27,3 +27,24 @@ jobs:
...
```
Comment (bring-shrubbery) [inline .github/workflows/test.action.yml:50]: I'll add this new line later, so don't worry about it :)

**Review by bring-shrubbery — APPROVED** [review]
Gonna merge for velocity sake :D

**Pull request merged**